pkg/omf/functions/cli/omf.cli.theme.fish

```diff
@@ -21,7 +21,7 @@ function omf.cli.theme -a name
     end
   case '*'
     echo (omf::err)"Invalid number of arguments"(omf::off) >&2
-    echo "Usage: $_ "(omf::em)"$argv[1]"(omf::off)" [<theme name>]" >&2
+    echo "Usage: $_ "(omf::em)"$argv[0]"(omf::off)" [<theme name>]" >&2
```

Comment:
Have you tried this code? It won't work as list indices start at 1 in fish. Please, check the [docs](https://fishshell.com/docs/current/language.html?highlight=indices+start#lists).

An accurate fix could be:
```suggestion
    echo "Usage: $_ "(omf::em)"t"(omf::off)"heme [<theme name>]" >&2
```

Reply:
My fault. I put in this change rather speedily and haven't really coded for awhile, so my knowledge is rusty. I'll commit the suggested change.

Reply:
Yeah, It's really confusing when pretty much all languages start indexing arrays/list at `0`. Lua is the same and it always gets me.

BTW, I noticed that none of the tests failed back then. Would you be willing to add tests for that?

Reply:
Missed this part. Yes I can do that.